client/state/selectors/can-site-view-atomic-hosting.js

```diff
@@ -35,6 +36,10 @@ export default function canSiteViewAtomicHosting( state ) {
 		return false;
 	}
 
+	if ( ! isSiteOnPaidPlan( state, siteId ) ) {
```

Comment:
Only business and eCommerce plans are available for Atomic sites, so we could do a more restrict check. Otherwise the `/hosting-admin` section would be visible for Atomic sites under a premium plan.

Reply:
Thanks, didn't think of that - have added a new selector to do a more precise check.